### PR TITLE
removed italics and fixed menu formatting

### PR DESCRIPTION
**Pull request by emilybrown1**
Part of a quality review of HDRP docs.

Removed italics and fixed menu formatting

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md [hunk 1]
```diff
@@ -25,9 +25,9 @@ Path tracing shares the general requirements and setup as other ray tracing effe
...
-4. If the image does not converge over time, select the drop-down next to the effect toggle and enable Always Refresh.
+2. In the Inspector, select **Add Override** > **Ray Tracing** > **Path Tracing**.
+3. In the Inspector for the Path Tracing Volume Override, check the Enable option. If you don't see the Enable option, make sure your HDRP Project supports ray tracing. For information on setting up ray tracing in HDRP, see [getting started with ray tracing](Ray-Tracing-Getting-Started.md). This switches HDRP to path-traced rendering and you should initially see a noisy image that converges towards a clean result.
+4. If the image doesn't converge over time, in the Scene view toolbar, select the drop-down next to the effect toggle and enable Always Refresh.
 
 ![](Images/RayTracingPathTracing3.png)
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md:32]: This screenshot needs updating.

File: com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md [hunk 2]
```diff
@@ -67,23 +67,23 @@ For that reason, some parameters have no effect in path tracing, while others be
 
 ### Refraction model
```
Comment (Vic-Cooper) [inline com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md:68]: Hm. I re-wrote this section and it should have been merged. I'll go investigate this.

**Review by Vic-Cooper — APPROVED** [review]
Thi page needs a lot of work but I've done most of it already :D AWe just need to figure out why it's not actually pushed... I'll approve this but be careful of merge conflicts.

**Review by sebastienlagarde — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
The other PR is this: https://github.com/Unity-Technologies/Graphics/pull/6494. We have a meeting with Vic to talk about it, but it will need rewording. For now I will merge this one as it ahve small adjustement.

**Pull request merged**